src/command/backup/backup.c

```diff
@@ -1182,6 +1183,9 @@ backupJobResult(
             // Else file was copied so update manifest
             else
             {
+                // Increment size of files copied
+                *sizeCopied += copySize;
+
```

Comment:
I think this will also need to be done at 1172 if we want the log output in backup to match what info outputs. That case means that a file existing before a resume is still usable.

Reply:
will add